src/streamlink/plugins/pixiv.py

```diff
@@ -94,7 +94,7 @@ def _get_streams(self):
         if login_username and login_password:
             self.logger.debug("Attempting login as {0}".format(login_username))
             if self._login(login_username, login_password):
-                self.logger.info("Successfully logged in as {0}".format(login_username))
+                self.logger.info("Logged in as {0}".format(login_username))
```

Comment:
it's just a copy and paste https://github.com/streamlink/streamlink/search?q=%22Successfully+logged+in+as+%7B0%7D%22&unscoped_q=%22Successfully+logged+in+as+%7B0%7D%22